src/store/actions.ts

```diff
@@ -297,11 +297,12 @@ export const setAddressSuggestions: ActionIF = ({ commit }, addressSuggestions:
 
 // FUTURE: not an action - move it to another module?
 export const fetchCorpNum = async ({ getters }, corpNum: string): Promise<any> => {
-  // NB: MRAS search should have been done on first page for xpro
-  // NB: COLIN search should have been done on first page for BC
-  if (getters.getShowCorpNum === CorpNumRequests.MRAS) {
-    return checkMRAS({ getters }, corpNum)
-  }
+  // NB: MRAS search was done on first page
+  // if (getters.getShowCorpNum === CorpNumRequests.MRAS) {
+  //   return checkMRAS({ getters }, corpNum)
+  // }
```

Comment:
Should be deleted but I think there's another flow that does a MRAS search so I want to wait to see if it needs this.